gateway-export-plugin/src/test/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/loader/CassandraConnectionLoaderTest.java

```diff
@@ -6,99 +6,85 @@
 
 package com.ca.apim.gateway.cagatewayexport.tasks.explode.loader;
 
-import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentTools;
-import com.ca.apim.gateway.cagatewayexport.tasks.explode.bundle.entity.CassandraConnectionEntity;
-import com.google.common.base.Joiner;
-import com.google.common.collect.ImmutableMap;
-import org.junit.jupiter.api.Test;
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
-
-import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BuilderUtils.buildAndAppendPropertiesElement;
-import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BundleElementNames.*;
-import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.*;
-import static java.security.Security.getAlgorithms;
-import static org.junit.jupiter.api.Assertions.*;
-
 class CassandraConnectionLoaderTest {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add some tests to this class. ([squid:S2187](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2187))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=122&issues=AWblzesfSgN4uCclhZKb&open=AWblzesfSgN4uCclhZKb)

[ISSUE_KEY]: AWblzesfSgN4uCclhZKb